addons/binding/org.openhab.binding.gpstracker/README.md

```diff
@@ -138,9 +156,13 @@ Thing gpstracker:tracker:EX   "EX tracker" [trackerId="EX"] {
 Location	locationEX	"Location"		{channel="gpstracker:tracker:1:lastLocation"}
 DateTime	lastSeenEX	"Last seen"		{channel="gpstracker:tracker:1:lastReport"}
 Number		batteryEX	"Battery level"		{channel="gpstracker:tracker:1:batteryLevel"}
+Number		accurayEX	"GPS Accuracy [%d m]"		{channel="gpstracker:tracker:1:gpsAccuracy"}
```

Comment:
Ok

Reply:
Two minors maybe:
* accuracyEX instead of accurayEX	
* it remains a little unclear that transition messages will only ever work when generated from external apps. While that's mentioned, it isn't apparent to laymen that the 'Tracker Location Status' paragraph only applies to working with external apps. 

Other than that I'm fine with the docs now.

Reply:
Well the 'Tracker Location Status' paragraph is valid for external and distance channel regions as well. Apparently it was not clear enough so I've made some changes here. Pls check.

Reply:
@mstormi 
Accuracy channel uses QuantityType updated in meters. Length specifies the dimension of this quantity. See [this](https://www.openhab.org/docs/concepts/units-of-measurement.html) for details.